CPAC/registration/tests/test_ants_registration_connector.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from CPAC.registration.registration import ANTs_registration_connector
+
+
+class AttrDict(dict):
+    def __getattr__(self, item):
+        value = self[item]
+        if isinstance(value, dict):
+            return AttrDict(value)
+        return value
```

Comment:
Why create a custom class for this test and not just use

```suggestion
from CPAC.utils.configuration import Configuration
```

?